demos/python_demos/face_recognition_demo/face_recognition_demo.py

```diff
@@ -390,11 +392,12 @@ def run(self, args):
     @staticmethod
     def open_input_stream(path):
         log.info("Reading input data from '%s'" % (path))
-        if path == 'cam':
-            stream = 0
-        else:
+        stream = path
+        try:
+            stream = int(path)
+        except ValueError:
             assert osp.isfile(path), "Input file '%s' not found" % (path)
-            stream = path
+            pass
         return cv2.VideoCapture(stream)
```

Comment:
There is no need in `pass`.

Or the check of file existence can be removed. `VideoCapture` can work not only with numbers and local files. For example, via `rtsp` protocol. I think it's better to check if `VideoCapture` `isOpened`.